dotCMS/src/integration-test/java/com/dotcms/datagen/FieldDataGen.java

```diff
@@ -26,6 +27,7 @@
     private boolean unique = Boolean.FALSE;
     private String contentTypeId;
     private String relationType;
+    private DataTypes dataType;
```

Comment:
Codacy found an issue: [Field dataType has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6354090)